packages/create-nx-workspace/src/utils/package-manager.ts

```diff
@@ -79,6 +81,12 @@ export function getPackageManagerCommand(
         globalAdd: 'npm i -g',
         getRegistryUrl: 'npm config get registry',
       };
+    case 'bun':
+      return {
+        install: 'bun install --silent --ignore-scripts',
+        exec: 'bunx',
+        globalAdd: 'bun install -g',
```

Comment:
Could you please add a comment here and elsewhere where we define the `bun` PackageManager linking to the issue which tracks the `config` command please?

Reply:
Are you on about the issue relating for the getRegistryUrl?  I'll see if there is an issue for it.